sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -133,7 +133,11 @@ public void execute(SensorContext context) {
 
     Map<File, Integer> numberOLinesOfCode = new HashMap<>();
 
-    analyseFiles(context, phpAnalyzer, inputFiles, progressReport, numberOLinesOfCode);
+    try {
+      analyseFiles(context, phpAnalyzer, inputFiles, progressReport, numberOLinesOfCode);
+    } catch (CancellationException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)